src/Client/Response/UserGroupValidationResponse.php

```
@@ -22,17 +22,17 @@
  */
 class UserGroupValidationResponse extends Response
 {
-    private bool $isGroup;
+    private ?bool $isGroup = null;
```

I think better if this is not nullable. It is simply a bool and will be set in constructor. Otherwise unset.

---

wich one, false or true? Both can be wrong, as we don't know. As I explained in the other PR, I would rather throw an exception and be sure the value is set correct

---

yeah, you are right, thanks!